receiver/mongodbreceiver/testdata/scraper/expected.json

```diff
@@ -21,8 +21,8 @@
                                     }
                                  }
                               ],
-                              "startTimeUnixNano": "1659372940162070000",
-                              "timeUnixNano": "1659372940162165000"
+                              "startTimeUnixNano": "1667524471740209000",
```

Comment:
There don't appear to be any meaningful changes in this file, so please revert them.